### ISSUE TITLE
toolkit license, GPL compatibility, and Drupal.org library whitelist

### ISSUE THREAD

**Issue by ghost** [developerforce/Force.com-Toolkit-for-PHP#12]
tl;dr:
can we distribute the toolkit under a [GPL-compatible](http://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses) license?

This issue came about when the [Drupal.org Library Packaging Whitelist](http://drupal.org/project/drupalorg_whitelist) project was formed. The project aims to ease integration with third party packages by allowing them to be included and distributed on drupal.org.

The catch is that, to be whitelisted, any third-party package must be licensed under a [GPL-compatible](http://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses) scheme.

While the license in the toolkit appears to be GPL-compatible, it is an unnamed license (and therefore not listed on gnu.org) and has been rejected for whitelisting (see http://drupal.org/node/1404212 ).

What do you think about changing the toolkit license to a similar, explicitly GPL-compatible license, so that the package may be included on drupal.org? 

This package is used in a number of Drupal projects, including [Salesforce Sutie](http://drupal.org/project/salesforce). As long as I'm maintainer of the Salesforce Suite, I'll take personal responsibility for keeping the drupal.org package in synch with the version on github.com
